cmd/root.go

```diff
@@ -8,7 +8,21 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/Pallinder/go-randomdata"
+	"github.com/segmentio/analytics-go"
 	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/pkg/analyticsutil"
+)
+
+const (
+	analyticsEndpoint = "https://analytics.starport.cloud"
+	analyticsKey      = "pWSXBMIF3tQsHTtA63Lb63zAfIA80Bhy"
```

Comment:
@fadeev let's add the key you created here it has to be public anyway. in future we can deliver it from CI if needed.